python3-aws-regions.spec

```diff
@@ -15,43 +15,42 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+%define python python
+%{?sle15_python_module_pythons}
 
-Name:           python3-aws-regions
+Name:           python-aws-regions
 Version:        0.1.2
 Release:        0
 Summary:        API for retrieving the most up-to-date AWS region list
 License:        MIT
 URL:            https://github.com/SUSE-Enceladus/aws-regions
 Source:         https://files.pythonhosted.org/packages/source/a/aws-regions/aws-regions-%{version}.tar.gz
 BuildRequires:  python-rpm-macros
-BuildRequires:  python3-requests
-BuildRequires:  python3-pytest
-BuildRequires:  python3-vcrpy
```

Comment:
vcr  in `BuildRequires` is missing, but it's still imported in `tests\test_endpoints.py`. Is this intended?

Reply:
Not intended. Was testing prior to vcrpy being packaged so dropped it from the spec.

Reply:
Fixed and rebased